README.md

```diff
@@ -2,13 +2,13 @@
 
 # Essencium
 
-[![Build](https://github.com/Frachtwerk/essencium-frontend/actions/workflows/build.yml/badge.svg)](https://github.com/Frachtwerk/essencium-frontend/actions/workflows/build.yml)
+[Build](https://github.com/Frachtwerk/essencium-frontend/actions/workflows/build.yml)
 
 [Preview](https://staging.essencium.dev)
 |
 [Docs](https://docs.essencium.dev)
 
-![Essencium Logo](./packages/app/src/img/web/icon-192.png)
+<img src="./packages/app/public/img/web/logotype_400x100px.svg" alt="Essencium Logo" width="300"/>
```

Comment:
You can also write this in markdown format to be consistent like the corrupt line was but with a wrong path to the logo.

Reply:
Thought the logo is a bit huge then but changed it. :)